docs/UserGuide.md

```diff
@@ -283,7 +285,7 @@ Deletes the specified event from the event list.
 
 Format: `delete -e INDEX`
 
-* Deletes the event at the specified `INDEX`.
+* Deletes the event of the specified `INDEX`.
```

Comment:
i think this may not be correct as it implies that an INDEX is tied to a specific contact.